### DISCUSSION TITLE
Is there a way to handle .pcss extension files as postcss?

### DISCUSSION THREAD

**Discussion by NeroMorto** [web-infra-dev/rsbuild#2765] [Q&A] [answered]
Now I'm getting error:
You may need an appropriate loader to handle this file type. **/*.pcss

**Comment by chenjiahan** [comment, marked as answer]
.pcss is not a commonly used ext. It is recommended to using.css instead

**Reply by NeroMorto** [reply]
I know( But it what it is. There are many many .pcss files in project, which contains postcsss and .css with "native" css.
Tried several options, but nothing is working properly.
Thx for reply anyway.

**Reply by chenjiahan** [reply]
Actually it is very easy to support `.pcss` in Rsbuild, just adding the following config to modify the CSS rule:

```ts
export default defineConfig({
  tools: {
    bundlerChain(chain, { CHAIN_ID }) {
      chain.module.rule(CHAIN_ID.RULE.CSS).test(/\.p?css$/);
    },
  },
});
```

**Reply by NeroMorto** [reply]
Wow! this worked like a charm! I'm somehow missed this section in docs(
Thank you very mach!
